radiography/src/main/java/radiography/Radiography.kt

```diff
@@ -0,0 +1,221 @@
+package radiography
+
+import android.annotation.TargetApi
+import android.content.res.Resources.NotFoundException
+import android.os.Build.VERSION_CODES.CUPCAKE
+import android.view.View
+import android.view.ViewGroup
+import android.view.WindowManager
+import android.widget.Checkable
+import android.widget.TextView
+import radiography.Radiography.scan
+
+/**
+ * Utility class to scan through a view hierarchy and pretty print it to a [String].
+ * Call [scan] or [View.scan].
+ */
+object Radiography {
```

Comment:
Why have this object at all, vs just having a top-level function?

Reply:
namespace polluting, ie if we do top level function then anywhere someone types `s` we'll start showing `scan` ?

Reply:
And now when anyone types `r` they'll see `Radiography`. I don't feel strongly here though, up to you.